core/qthelper.h

```diff
@@ -11,6 +11,7 @@ struct picture;
 // 1) Types
 
 enum inertgas {N2, HE};
```

Comment:
Not a comment on this PR, just an idle remark:
Why is `intergas` defined here and not in `core/gas.h`?

Reply:
It's a very good question.